resources/assets/components/pages/CampaignsPage/CampaignFilters/CauseFilter/CauseFilter.js

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import causes from './CauseVariables';
+import Button from '../../../../utilities/Button/Button';
+
+const CauseInputs = ({ causeChecked, causeName, causeValue, handleSelect }) => (
+  <>
+    <label className="flex justify-start pb-2" htmlFor={causeName}>
+      <input
+        name={causeValue}
+        id={causeValue}
+        type="checkbox"
+        value={causeValue}
+        onClick={handleSelect}
+        checked={causeChecked}
+      />
+      <span className="pl-4">{causeName}</span>
+    </label>
+  </>
```

Comment:
You can get rid of this wrapping fragment, since it's just a single `label` element inside it:

```suggestion
  <label className="flex justify-start pb-2" htmlFor={causeName}>
    <input
      name={causeValue}
      id={causeValue}
      type="checkbox"
      value={causeValue}
      onClick={handleSelect}
      checked={causeChecked}
    />
    <span className="pl-4">{causeName}</span>
  </label>
```